### PR TITLE
Provide instructions for advanced_industrial_ci

### LINKED ISSUE #29: Reducing build time for VTK 7 and PCL trunk
From [the maling list](https://groups.google.com/forum/#!topic/swri-ros-pkg-dev/rpGkmKlBebs) by @VictorLamoine

>> I would like to use indusrial_ci for the bezier project.
>> However the dependencies are VTK 7 and PCL trunk so I need to compile/install those on the Travis server.
>> This takes time and exhausts the available time for "free" builds.
>> 
>> See for example:
>> https://travis-ci.org/InstitutMaupertuis/bezier/jobs/123141945
>> 
>> I reduced the PCL build to a minimal build but it still takes a lot of time.
>> There are some PPA for VTK nightly but no PPA for PCL trunk.
>> 
>> Is there a way to speed up this? Providing binaries?

@shaun-edwards 

> Do you need to build PCL/VTK every time or would weekly/monthly binaries be OK?  If so, then I could imagine that you modify the Travis script to download the binaries/debians from some repo (PPA, github, ftp, etc...) and install them.  You would still be on the hook to recompile PCL/VTK every so often, but I'm guessing you aren't worried about those libraries causing issues.

### PR DESCRIPTION
**Pull request by VictorLamoine**
Fixes #29 
I think the scripts are good enough, everything works across several repositories.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,6 +30,10 @@ FAQ
 
   A- The `industrial_ci` still provides valuable checks; it ensures if your package builds without issues. Also installation rules if you define. Just as a headsup that making test cases are highly recommended as your ear may hurt.
 
+- Q- My package uses a custom Point Cloud Library (PCL) version or the [`industrial_calibration`](https://github.com/ros-industrial/industrial_calibration) package, how do I make build work?
```
Comment (130s) [inline README.rst]: This is `rst` file so link needs to be something like:

    `industrial_calibration <https://github.com/ros-industrial/industrial_calibration>`_

File: README.rst [hunk 2]
```diff
@@ -30,6 +30,10 @@ FAQ
...
+
+  A- You can check [`advanced_industrial_ci`](https://github.com/InstitutMaupertuis/advanced_industrial_ci), this repository provides scripts and binaries for PCL 1.8.0 and the Ceres solver.
```
Comment (130s) [inline README.rst]: I think including links to existing samples will be very helpful.

Comment (130s) [inline README.rst]: Also,

> this repository

readers might get lost which repository this line refers to (`industrial_ci` or `advanced_industrial_ci`).
You might simply want to replace:

- - this repository
- + which

**Comment by 130s** [comment]
I left minor comments. If you could address those this will be a nice external enhancement.

[A test is failing](https://travis-ci.org/ros-industrial/industrial_ci/jobs/135387111), which is [negligible for now](https://github.com/ros-industrial/industrial_core/pull/144#issuecomment-223186764).

**Comment by VictorLamoine** [comment]
All fixed!

**Pull request merged**